source/index.html.md.erb

```diff
@@ -25,6 +25,7 @@ It is inspired by the [GDS Way](https://gds-way.cloudapps.digital) and the
 ### Building software
 
 * [Licensing software or code](documentation/standards/licencing-software-or-code.html)
+* [Metrics](documentation/standards/metrics.html)
```

Comment:
I think it should be a subsection of paas as it's quite specific to paas. We only use prometheus in paas I believe.